jni_gen/third_party/ApiSummarizer/src/main/java/com/github/hegde/mahesh/apisummarizer/util/SkipTypeException.java

```diff
@@ -0,0 +1,7 @@
+package com.github.hegde.mahesh.apisummarizer.util;
+
+public class SkipTypeException extends RuntimeException {
```

Comment:
Should this extend SkipException?

Reply:
I think I can do away with SkipMethodException and SkipClassException, because realistically only handler will know to which extent to skip.